app/services/list_republisher.rb

```diff
@@ -0,0 +1,61 @@
+class ListRepublisher
+  def republish_tags(tags)
+    done = 0
+    tags.find_each do |tag|
+      next if tag.lists.blank?
+
+      update_groups(tag)
+      republish_tag(tag)
+
+      done += 1
+
+      if (done % 100).zero?
```

Comment:
What's the significance of `% 100` 

Could you use each_with_index instead of the numerator? Or am I missing something - which is VERY LIKELY tbh!

Reply:
Blindly copied from https://github.com/alphagov/collections-publisher/blob/main/app/services/tag_republisher.rb
I don't have special feelings about `% 100` 😆 and wouldn't question Tijmen.
To me it serves the purpose of outputting something to the console to indicate that it's doing something.

Reply:
I guess it's to reduce the amount of log output – so you'd end up with:

```
0 completed...
100 completed...
200 completed...
```

Rather than a (much noisier):

```
0 completed...
1 completed...
2 completed...
3 completed...
```